scripts/get-usage/deduplicate-components.ts

```diff
@@ -0,0 +1,42 @@
+const componentNameChanges: Record<string, string | undefined> = {
+	'@guardian/src-ed-button/Button':
+		'@guardian/source-react-components-development-kitchen/EditorialButton',
+	'@guardian/src-ed-button/LinkButton':
+		'@guardian/source-react-components-development-kitchen/EditorialLinkButton',
+};
+
+export const deduplicateComponents = (components: string[]): string[] => {
+	const oldComponents: string[] = [];
+	const newComponents: string[] = [];
```

Comment:
I'm wary that the `old` and `new` prefixes here are a bit ephemeral. I wish we had given the "new world" an actual name instead of `new` or `v4`. I think what we mean here is `src` vs `source`, so maybe these should be the prefixes?